cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/client/AzureClient.java

```diff
@@ -976,6 +980,33 @@ public String getVaultNameFromEncryptionKeyUrl(String encryptionKeyUrl) {
         return null;
     }
 
+    public Map<String, List<String>> getAvailabilityZones(String region) throws ProviderAuthenticationFailedException {
+        return handleException(() -> {
+            Map<String, List<String>> zoneInfo = new HashMap<>();
+            if (StringUtils.isEmpty(region)) {
+                return zoneInfo;
+            }
```

Comment:
please add else

Reply:
We are returning Empty map if region is not given so there is no else here. If region is given then API will be called to fetch the availability zones for all instance type.

Reply:
please add else so the code structure indicates at first blink there is an exit point here